src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ResourceMonitoringOptions.cs

```diff
@@ -97,4 +98,15 @@ public partial class ResourceMonitoringOptions
         UrlFormat = DiagnosticIds.UrlFormat)]
     [TimeSpan(MinimumCachingInterval, MaximumCachingInterval)]
     public TimeSpan MemoryConsumptionRefreshInterval { get; set; } = DefaultRefreshInterval;
+
+    /// <summary>
+    /// Gets or sets a value indicating whether CPU metrics are calculated via cgroup CPU limits instead of Host CPU delta.
+    /// </summary>
+    /// <value>
+    /// The default value is true.
+    /// </value>
+#pragma warning disable CA1805 // Do not initialize unnecessarily
+    [Experimental(diagnosticId: DiagnosticIds.Experiments.ResourceMonitoring, UrlFormat = DiagnosticIds.UrlFormat)]
+    public bool CalculateCpuUsageWithoutHost { get; set; } = false;
```

Comment:
```suggestion
    public bool CalculateCpuUsageWithoutHost { get; set; }
```
as a result, there won't be the C1805 warning generated